src/main/scala/org/camunda/feel/impl/builtin/BooleanBuiltinFunctions.scala

```diff
@@ -70,4 +71,11 @@ object BooleanBuiltinFunctions {
     }
   )
 
+  private def isBlankFunction = builtinFunction(
+    params = List("string"),
+    invoke = {case List(ValString(string)) =>
+      ValBoolean(string.isBlank)
+    }
+  )
```

Comment:
:x: Since `is blank()` is a function for a string argument, we should add the function to [StringBuiltinFunctions](https://github.com/camunda/feel-scala/blob/main/src/main/scala/org/camunda/feel/impl/builtin/StringBuiltinFunctions.scala). 

`BooleanBuiltinFunctions` is for functions with a boolean argument.